src/providers/test-result/test-result.ts

```diff
@@ -127,14 +131,30 @@ export class TestResultProvider {
     return of(ActivityCodes.PASS);
   }
 
+  private calculateCatEUAM2AndSubCategoryTestResult = (
+    category: TestCategory,
+    testData: TestDataAM2,
+  ): Observable<ActivityCode> => {
+    if (this.faultCountProvider.getDangerousFaultSumCount(category, testData) > 0) {
+      return of(ActivityCodes.FAIL);
+    }
+    if (this.faultCountProvider.getSeriousFaultSumCount(category, testData) > 0) {
+      return of(ActivityCodes.FAIL);
+    }
+    if (this.faultCountProvider.getDrivingFaultSumCount(category, testData) >= 6) {
```

Comment:
This should be greater then 10 faults for a fail also i think @faggc covers this part?

Reply:
Yes, to my knowledge this should be "> 10" as per the description of my task.